File: app/src/main/java/com/mavenkalabs/adskipper/MainActivity.java
package com.mavenkalabs.adskipper;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.mavenkalabs.adskipper.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/com/mavenkalabs/adskipper/MainActivity.kt
package com.mavenkalabs.adskipper

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.mavenkalabs.adskipper.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.getRoot())
    }
}